caab-service/src/main/java/uk/gov/laa/ccms/api/mapper/IgnoreUnmappedMapperConfig.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.laa.ccms.api.mapper;
+
+import org.mapstruct.MapperConfig;
+import org.mapstruct.ReportingPolicy;
+
+/**
+ * Mapper config to ignore unmapped mapper warnings.
+ */
+@MapperConfig(unmappedTargetPolicy = ReportingPolicy.IGNORE)
```

Comment:
Does this mean the config would be applied to all Mappers by default?
I think while writing a mapper it's good to be told if something isn't mapped.